.github/workflows/ci.yml

```diff
@@ -0,0 +1,40 @@
+on: [push, pull_request]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    services:
+      postgres:
+        image: postgres:latest
+        env:
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: postgres
+          POSTGRES_DB: foreign_travel_content_checker_test
```

Comment:
I'm not sure you actually need much of this as per: https://docs.publishing.service.gov.uk/manual/test-and-build-a-project-with-github-actions.html#gov-uk-rails-application-with-postgres-redis-yarn-and-gov-uk-content-schemas-dependencies

Reply:
Thanks - removed